packages/web/src/scss/components/Modal/_Modal.scss

```diff
@@ -78,6 +78,20 @@
     margin-bottom: theme.$header-spacing;
 }
 
+.Modal__footer {
```

Comment:
Please add these classes to `index.html` example. Eg. please create an example where the changes you made will be visible. Thanks.

Reply:
I am not able to find the changes in the demo. Is it actually resolved?